libs/scripts/scripts.js

```diff
@@ -119,12 +120,14 @@ const locales = {
   sea: { ietf: 'en', tk: 'hah7vzn.css' },
 };
 
+const unavMeta = getMetadata('universal-nav')?.trim();
+
 const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
   imsClientId: 'milo',
-  imsScope: 'AdobeID,openid,gnav,pps.read,firefly_api',
```

Comment:
For the milo token, this should be irrelevant since it has the correct scopes, no?

Reply:
Indeed. The use-case I was thinking of is when a developer would overwrite the client ID to something that suits their testing purposes. That client ID might not be onboarded to these additional scopes and it might not be obvious why IMS is no longer working; not sure if scopes are the first culprit for everyone in such situations. So I decided to keep the logic here as well. However, you do have a point - when everyone will be using the Universal Nav, then basing scopes on the feature would no longer be relevant.

With this in mind, would you still just leave the static scopes or have the dynamically built one until the Universal Nav becomes standard?

Reply:
What if we do something like:
```js
const config = {
    // some existing config
    imsScope: 'AdobeID,openid,gnav',
    // other config
};

const unavMeta = getMetadata('universal-nav')?.trim();
if (unavMeta && unavMeta !== 'off') config.imsScope += ',pps.read,firefly_api';
```
I personally find this more readable and would allow folks to add more default scopes _before_ the ones needed for universal navigation.

Reply:
The reason I didn't go for this is that I wanted the scopes to be grouped in a single location. Otherwise one would look and see just 3 scopes, while on a UNav-enabled page they see 5. They'd have to go back to the file and check additional logic, whereas my suggested change has everything in one line so the additional logic is clear from the begining.

Reply:
> With this in mind, would you still just leave the static scopes or have the dynamically built one until the Universal Nav becomes standard?

Should we not set any config at all and consume the standard config? That way we consume them in a dog-food manner as a milo college project would

Reply:
Had a look over the history of this line and saw that @narcis-radu first introduced it [here](https://github.com/adobecom/milo/pull/736/files#diff-12979217e83810ba83417c3e7b030afd400eff7c83cd6390d9f9de3a337618ec). So with the current way we load IMS, I'd say it's safe to remove the declaration from here altogether and just rely on the sensible default, as you're suggesting.